source/creating_content/01_creating_content_page/managing_page_security.rst

```diff
@@ -0,0 +1,118 @@
+
+.. admonition:: Caution
+   :class: caution
+
+   If a user has access to a content page but which was created with datasets that user isn't allowed to access, all related visualizations will not load. However, the user will be able to read all other information displayed on the content page, such as text.
+
+Managing page permissions for users and groups
+----------------------------------------------
+
+.. admonition:: Important
+   :class: important
+
+   Users and groups must be added in the Users & Groups section beforehand, otherwise they can't be found in a page security management. No matter the access policy or the permission, permissions can indeed be given to users invididually or through groups of users, but both users and groups must already be listed in the Users & Groups section of the back office (where users are invited to the portal and where groups are created).
+
+There are 2 permissions available in the security management of a content page:
+
+- *Browse page*: it allows the user/group of users to access the page and its content. This permission is set by default for all users added to the security management section of the page.
+- *Edit page*: it allows the user/group of users to access the back office to edit the content of the page.
+
+These permissions have different impacts depending on the chosen access policy.
+
+- *Browse page* permission:
+
+  - For a private page, the *Browse page* permission gives access to that page, hence the importance of adding users to the security management section, otherwise nobody (except users with the general domain-level *Browse all pages*) can access the page and its content.
+  - For a public page, the *Browse page* permission doesn't change much to the access security of that page because it is already accessible by anybody who has access to the domain.
+
+- *Edit page* permission: this permission is relevant for both private and public pages and gives edition rights on the page to any user/group of users added to the security management section of the page and granted with this *Edit page* permission.
+
+Adding and deleting users or groups of users from a page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Adding users or groups to a page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to Security > User permissions or Security > Group permissions, depending on whether a user or a group will be added.
+2. Click the **Add user/group permissions** button at the bottom of the table.
+3. Select one or more users/groups of users from the list. A search bar is also available to search a user/group by name.
+4. (optional) If all added users/groups are going to be granted the *Edit page* permission as well as the default *Browse page* permission, it is possible to grant that permission now. Otherwise permissions can be granted afterward. To already grant the *Edit page* permission, at the top of the "Add user/group permissions" window, in the "Grant these permissions section":
```

Comment:
I think there's a typo, it should be `"Grant these permissions" section` instead of `"Grant these permissions section"`